### ISSUE TITLE
[BUG] Accessible name different from on-screen label

### ISSUE THREAD

**Issue by adarshrema** [microsoft/accessibility-insights-windows#707]
where: Settings > Connection
Issue: Assistive Technology(AT) users are presented with details different from on-screen lable for issue filing link text box
Repro steps:
1. Open NVDA screen reader 
2. Navigate to Settings->connection in Accessibility Insights for Windows 
3. Select Azure Boards radio button 
4. Navigate to the input box under “Enter desired Azure Boards link” using keyboard and compare NVDA announcement to on-screen label

Expected behavior:
Update accessible name of the control to be identical to on-screen label so AT users are presented with the same information that is available visually

**Comment by DaveTryon** [comment]
@adarshrema, per our conversations, the AT experience still lacks an equivalent of the placeholder text, but it's an intentional compromise to avoid sensory overload from JAWS or NVDA repeating the prompt and the placeholder twice. Change is in the current canary build. Please regress/close as appropriate.

**Comment by adarshrema** [comment]
Validated fix in Canary
